src/actions/textobject.ts

```diff
@@ -640,3 +641,219 @@ class InsideIndentObjectBoth extends IndentObjectMatch {
   includeLineAbove = true;
   includeLineBelow = true;
 }
+
+abstract class SelectArgument extends TextObjectMovement {
+  modes = [Mode.Normal, Mode.Visual];
+
+  private static openingDelimiterCharacters(): string[] {
+    return configuration.argumentObjectOpeningDelimiters;
+  }
+  private static closingDelimiterCharacters(): string[] {
+    return configuration.argumentObjectClosingDelimiters;
+  }
+  private static separatorCharacters(): string[] {
+    return configuration.argumentObjectSeparators;
+  }
+
+  protected selectAround = false;
```

Comment:
Can you add a comment here describing the difference between "inner" and "around"?